test/OpenSTUtility.js

```diff
@@ -426,33 +427,29 @@ contract('OpenSTUtility', function(accounts) {
 				brandedTokenContract = new BrandedToken(brandedToken);
 				await brandedTokenContract.claim(accounts[0]);
 				await brandedTokenContract.approve(openSTUtility.address, redemptionAmount, { from: redeemer });
-	            result = await openSTUtility.redeem(checkBtUuid, redemptionAmount, 2, { from: redeemer });
+	            result = await openSTUtility.redeem(checkBtUuid, redemptionAmount, 2, lockR.l, { from: redeemer });
 	            redemptionIntentHash = result.logs[0].args._redemptionIntentHash;
 		    })
 
 			it('fails to process if redemptionIntentHash is empty', async () => {
-	            await Utils.expectThrow(openSTUtility.processRedeeming("", { from: redeemer }));
-			})
-
-			it('fails to process if msg.sender is not redeemer or registrar', async () => {
```

Comment:
this test can be replaced with its counterpart : "fails to process when presented with wrong secret"